inventory/foreman.py

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env python
```

Comment:
Since i need to template the above file anyway, might as well pull this one down from the ansible repo.